Source/SwiftLintFramework/Rules/RuleConfigurations/FileNameNoSpaceConfiguration.swift

```diff
@@ -0,0 +1,33 @@
+public struct FileNameNoSpaceConfiguration: RuleConfiguration, Equatable {
+    public var consoleDescription: String {
+        return "(severity) \(severity.consoleDescription), " +
+            "excluded: \(excluded.sorted())"
```

Comment:
This should also include the suffix pattern.

Reply:
I used `file_name` rule as an example and didn't see suffix or prefix in the [consoleDescription](https://github.com/realm/SwiftLint/blob/20ba18c820602c29fe0e3c335adcc4f78bf6ca84/Source/SwiftLintFramework/Rules/RuleConfigurations/FileNameConfiguration.swift#L4). Is that a mistake?

Reply:
Yes that's a mistake. Would you mind adding that in a separate PR please?